### ISSUE TITLE
[MP4Box Feature Request] Remove "X" (seconds) from End of File

### ISSUE THREAD

**Issue by starkline** [gpac/gpac#817]
I wasn't sure if this is the proper place for such a thing, but the website seemed to indicate this is a good place to add an idea for a feature. Please, forgive me if this is the wrong place.

When using -splitx we have the ability to indicate a split from a specified point to the end of the file:
      (example: '-splitx 10:end')

My idea is to add the ability to specify a certain length to be excluded from the end of the file. In other words, to remove a certain number of seconds from the end of the file.
     (example: '-splitx 10:end5' would remove 5 seconds from the end of an input file.)

Just a thought! :)
Cheers, and thanks for one of the most useful tools on the planet!

**Comment by jeanlf** [comment]
done, thanks for the idea.

**Comment by starkline** [comment]
Cheers! This will be really handy.

**Comment by starkline** [comment]
Just a quick comment here in case someone else randomly searches Google and finds the thread.

I hadn't used this feature in some time, so I forgot the syntax and did a search for it. I recalled the syntax was slightly different from my example idea when it was initially implemented, so I knew where to look!

Working Example:
-splitx 0:end-5.5

That will remove 5.5 seconds from the end. Note the dash "-" between "end" and the integer. Easy enough to look at the code and determine this is needed, but in case someone doesn't know how, I decided I'd post this.
